### ISSUE TITLE
[Bug] File encoding issue when CLI takes arguments via file. e.g. az devops wiki page create , az devops invoke 

### ISSUE THREAD

**Issue by atbagga** [Azure/azure-devops-cli-extension#581]
**Describe the bug**
I cannot use utf-8 encoded file to create a wiki page

**To Reproduce**
Azure Cli Version: 2.0.63
Azure-Devops extension version: 0.6.0

1. Configure and login as usual
1. Create a file with content using powershell e.g.  echo "hello world" | Out-File -FilePath 'd:/a.txt'
1. Try to create a page with the file in #3 as content. 
e.g. az devops wiki page create --wiki myprojectwiki --path /mypage3 --file-path "d:/a.txt" 

Error - Page content looks like this - 
![image](https://user-images.githubusercontent.com/11888714/57219892-1b888000-7017-11e9-9041-f4130f60a3f6.png)

**Expected behavior**
The page should be created with correct content.

**Comment by atbagga** [comment]
Possible solutions: 

1. Mention the encoding as utf-8 in the  command help text and read the file as utf-8 else error out appropriately.
1. Allow a -encoding parameter with values like utf-8, utf-16, ascii etc and read the file appropriately. User will have option to use any encoding in the list.
```az keyvault secret set -h``` uses option #2 to allow users to enter the encoding type. See help for reference.

Sample help text from 'az keyvault secret set -h'

``` 

Content Source Arguments
    --encoding -e           : Source file encoding. The value is saved as a tag (`file-
                              encoding=<val>`) and used during download to automatically encode the
                              resulting file.  Allowed values: ascii, base64, hex, utf-16be,
                              utf-16le, utf-8.  Default: utf-8.
    --file -f               : Source file for secret. Use in conjunction with '--encoding'.
    --value                 : Plain text secret value. Cannot be used with '--file' or '--encoding'.
```

**Comment by SathishRanjan** [comment]
why cant we autodetect it? we do that in server side

**Comment by atbagga** [comment]
I did look for options to auto-detect and it seems correctly detecting the encoding at all times is impossible. 

There is nothing in the python standard library, though there are some other third party modules to auto detect but not with 100% accuracy (e.g. chardet library) and not without performance cost.

**Comment by atbagga** [comment]
@snankani @geverghe For triage. This got skipped in today's triage. 
Should we go with approach 2 suggested above?

**Comment by geverghe** [comment]
We should definitely start with approach 2. We can look at auto detection as part of next step. @SathishRanjan - could you help us understand how you are doing it server side?

**Comment by atbagga** [comment]
Fixed and will be available in next release 0.10.0. 
ETA of release: early next week.
